readme.txt

```diff
@@ -37,6 +37,10 @@ We're closely monitoring continued developments in the ad fraud space, and see t
 
 We plan on adding support for this file in the future! If you'd like to follow along with development or even help us get it done, please see [this GitHub issue](https://github.com/10up/ads-txt/issues/40).
 
+=== What about using this on a multisite setup? ===
+
+You can only have one ads.txt for a given domain or subdomain per the [ads.txt spec](https://iabtechlab.com/ads-txt/), it is site-by-site but if it’s a multisite subfolder install then only the main site will actually do anything.  Our recommendation is to only activate Ads.txt Manager per-site.
```

Comment:
```suggestion
=== Can I use this with multisite? ===

Yes! However, if you are using a subfolder installation it will only work for the main site. This is because you can only have one ads.txt for a given domain or subdomain per the [ads.txt spec](https://iabtechlab.com/ads-txt/).  Our recommendation is to only activate Ads.txt Manager per-site.
```